src/api/goods.ts

```diff
@@ -10,10 +10,13 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((good1, good2) => good1.name.localeCompare(good2.name))
```

Comment:
bad practice to use a number in variable naming, I think we can use `a` and `b` in sort method. 
It is normal so don't worry(but only for sort method)
```suggestion
      .sort((a, b) => a.name.localeCompare(b.name))
```